content/hardware/02.hero/boards/yun-rev2/tutorials/01.yun-download-os/content.md

```diff
@@ -0,0 +1,33 @@
+---
+title: 'Download GNU/Linux OS to Your Arduino Yún'
+description: 'Instructions for downloading the GNU/Linux OS to your board.'
+tags: 
+  - tutorial
+author: 'Arduino'
+hardware:
+  - hardware/03.hero/boards/yun-rev2
+software:
+  - ide-v1
+  - ide-v2
+  - web-editor
+  - iot-cloud
+---
+
+Here you can find the necessary download links to GNU/Linux OS to use with your Arduino Yún Rev2 board.
+
+## OpenWRT - Yún 1.5.3 Upgrade Image
+
+Download the latest stable GNU/Linux OS for your Yún. It makes your Yún more stable and feature rich.
+
+### Easy Installation Procedure (Recommended)
+
+Follow the steps in the [Yún Sys Upgrade tutorial](./yun-sys-upgrade).
+
+### Advanced Installation Procedure
+
+This procedure is only recommended to advanced users who wish to completely re-flash the Yún including its U-Boot bootloader. [These instructions](./yun-u-boot-reflash) on reflashing the base images are for reference only. Following them will void your Yún's warranty.
+
+<DownloadLink href="/openwrtyun/1/YunSysupgradeImage_v1.5.3.zip" title="Download" />

```

Comment:
I don't think this element renders at all. Maybe just turn this into a link?

Reply:
How do we do that? Just `<a href="/openwrtyun/1/YunSysupgradeImage_v1.5.3.zip" title="Download" />` ?

Reply:
I think just a regular link, like `[Download](https://downloads.arduino.cc/openwrtyun/1/YunSysupgradeImage_v1.5.3.zip)`, will be enough!